metron-interface/metron-alerts/src/app/alerts/alerts-list/tree-view/tree-view.component.ts

```diff
@@ -337,12 +343,67 @@ export class TreeViewComponent extends TableViewComponent implements OnChanges {
     });
   }
 
+  canCreateMetaAlert(count: number) {
+    if (count > 999) {
```

Comment:
Given the practical restriction detailed at https://issues.apache.org/jira/browse/METRON-1279, does this limit need to be lowered?  I've seen ~350 work without issues, personally.  Obviously, given the way the update works, I'm not sure we can ever guarantee a fixed limit, but even 500 seems really likely to blow up.

Reply:
Do we prevent users from adding enough alerts to go over the limit?  E.g. if I have 995 alerts and I try to add 20 more, will the UI prevent it?

Reply:
Ah ok, I was able to add 999 from my full-dev. What is your suggestion on the number of meta alerts 350 or 500?.

We are not validating the size when users try's to add few more alerts to meta alerts. If you confirm the possible number I will add it too.

Can I limit it to 350 for now?

Reply:
@james-sirota You'd know a bit more about the practical usage pattern than I would.  Do you have any input on this limit?  I'm guessing we could/should probably drop it a bit to give ourselves a bit more safety factor (e.g. 200?), assuming that fits a practical pattern.